### PR TITLE
chore: forward port ledger fixes

### PR DESCRIPTION
**Pull request by cmwaters**
- fix: signing with Ledger throws an empty error (#349)
- fix: restore Ledger support for MacOS 14 (#352)

### CODE REVIEW

**Review by evan-forbes — APPROVED** [review]

File: go.mod [hunk 1]
```diff
@@ -18,7 +18,7 @@ require (
 	github.com/cosmos/cosmos-sdk/db v1.0.0-beta.1
 	github.com/cosmos/go-bip39 v1.0.0
 	github.com/cosmos/iavl v0.19.6
-	github.com/cosmos/ledger-cosmos-go v0.12.2
+	github.com/cosmos/ledger-cosmos-go v0.13.2
```
Comment (rootulp) [inline go.mod:21]: 0.13.2  should work but in a follow up PR we should probably bump to https://github.com/cosmos/ledger-cosmos-go/releases/tag/v0.13.3

**Review by rootulp — APPROVED** [review]
What Cosmos SDK major release is `main` in this repo based on?

**Comment by cmwaters** [comment]
> What Cosmos SDK major release is main in this repo based on?

Still v0.46. It was cut from `releases/v0.46.x-celestia`. Actually I brought it up in slack that given we don't plan to break anything (i.e. we will use version based if statements) that it would be okay to just use the single branch and thus we don't need `main`

**Comment by rootulp** [comment]
oh awesome, that makes things easier to track. Does that mean we don't need this PR?

**Comment by cmwaters** [comment]
Closing since we are planning on deprecating the main branch

**Pull request closed without merging**